src/main/java/seedu/address/model/entry/Name.java

```diff
@@ -5,26 +5,30 @@
 import seedu.address.commons.exceptions.IllegalValueException;
 
 /**
- * Represents an Entry's name in the entry book.
- * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
+ * Represents an Entry's name in the entry book. Guarantees: immutable; is valid
```

Comment:
Should this be on the next line so the purpose summary of the class is on a line by itself?